packages/ERTP/src/paymentLedger.js

```diff
@@ -146,8 +149,10 @@ export const makePaymentLedger = (
       assertLivePayment(payment);
       const paymentBalance = paymentLedger.get(payment);
       assertAmountConsistent(paymentBalance, optAmount);
-      // Commit point.
-      paymentLedger.delete(payment);
+      fatalAssert.notThrows(() => {
```

Comment:
this is unnecessary because there's only one operation. This here would be better with a try/catch pattern because existing developers and tools would recognize it

Reply:
Agreed. Closed https://github.com/endojs/endo/pull/733